config/locales/ad_privacy_policy.en.yml

```diff
@@ -0,0 +1,40 @@
+en:
+  ad_privacy_policy:
+    show:
+      heading: "Privacy policy for Waste Carriers, Brokers and Dealers assisted digital"
```

Comment:
```suggestion
      title: "Privacy policy for assisted digital"
      heading: "Privacy policy for Waste Carriers, Brokers and Dealers assisted digital"
```

Reply:
This will add a unique title attribute to the page. At the moment we just have a generic Waste Carriers title. (Been busted during QA for this before!)

Reply:
Nice one! :D